globals.mock.js

```diff
@@ -0,0 +1 @@
+global.theme = {}
```

Comment:
maybe we can use the same theme as storybook (in .storybook/themes)

Reply:
Do we want to test themes in UTs as well ?